variables.tf

```diff
@@ -0,0 +1,41 @@
+variable "chart_version" {
+  description = "Chart version of the ingress controller"
+  type        = string
+}
+
+variable "namespace" {
+  description = "Namespace of the ingress controller"
+  type        = string
+}
+
+variable "host" {
+  description = "The hostname (in form of URI) of Kubernetes master"
+  type        = string
+}
+
+variable "cluster_ca_certificate" {
+  description = "PEM-encoded root certificates bundle for TLS authentication"
+  type        = string
+}
+
+variable "token" {
+  description = "Token to authenticate a service account"
+  type        = string
+}
+
+variable "lifecycle_ignore_changes" {
+  description = "Ignore lifecycle changes"
+  type        = list
+}
+
+variable "additional_values" {
+  description = "Additional values to pass to the helm chart"
+  type        = any
+  default     = {}
+}
```

Comment:
I think that the default value is an empty list (`[]`) and not an empty map (`{}`). The expected value for `values` property is a list of string. You can check the terraform documentation [here](https://registry.terraform.io/providers/hashicorp/helm/latest/docs/resources/release#values).

The `type` also could be converted into `list(string)`.

You can use [this](https://github.com/sparkfabrik/terraform-sparkfabrik-prometheus-stack/blob/main/variables.tf#L71) as an example.